README.md

```diff
@@ -95,6 +96,32 @@ if let foo = foo {
 
 _Rationale:_ Explicit `if let`-binding of optionals results in safer code. Force unwrapping is more prone to lead to runtime crashes.
 
+####Optional Chaining
+
+Optional chaining in Swift is similar to messaging `nil` in Objective-C, but in a way that works for any type, and that can be checked for success or failure.
+
+Use optional chaining if you want to assign a value to a property on an optional and you don’t plan on taking any alternative action if that optional is `nil`.
+
+**For Example:**
+
+```Swift
+let cell: YourCell = tableView.ip_dequeuCell(indexPath)
```

Comment:
How much of our own dog food is appropriate for this style guide?  This method in particular is pretty clear what its intent is, but should we assume the reader is familiar with IntrepidPursuits/swift-wisdom?